presto-tests/src/test/java/io/prestosql/connector/system/TestSystemConnector.java

```diff
@@ -100,6 +107,36 @@ public void tearDown()
         executor.shutdownNow();
     }
 
+    @Test
+    public void testRuntimeQueriesTimestamps()
+    {
+        ZonedDateTime timeBefore = ZonedDateTime.now();
+        computeActual("SELECT 1");
+        MaterializedResult result = computeActual("" +
+                "SELECT max(created), max(started), max(last_heartbeat), max(\"end\") " +
+                "FROM system.runtime.queries");
+        ZonedDateTime timeAfter = ZonedDateTime.now();
+
```

Comment:
We can do a sanity check to make the test failure reason obvious in case the system clock happens to be adjusted in between the calls
```java
assertThat(timeAfter).isAfterOrEqualTo(timeBefore);

Reply:
if the clock is adjusted, we still can fail, even if `timeAfter >= timeBefore`, so not sure how helpful it would be